src/adapter/src/catalog.rs

```diff
@@ -93,7 +93,8 @@ pub mod builtin;
 pub mod storage;
 
 pub const SYSTEM_CONN_ID: ConnectionId = 0;
-const SYSTEM_USER: &str = "mz_system";
+pub const SYSTEM_USER: &str = "mz_system";
+pub const HTTP_DEFAULT_USER: &str = "anonymous_http_user";
```

Comment:
I think this should start with `mz_` because that's guaranteed to be a protected username suffix.

Reply:
I don't think this is possible because the way this works is that `anonymous_http_user` doesn't exist unless someone tries to use the HTTP API with a username. Then we will create `anonymous_http_user` on the spot as a user Role. However we don't allow any user roles to be created with the prefix `mz_` because it's a reserved prefix.